codegen/lib/graphql_swift_gen.rb

```diff
@@ -221,9 +221,45 @@ def deserialize_value_code(field_name, expr, type, untyped: true)
       raise NotImplementedError, "Unexpected #{type.kind} argument type"
     end
   end
-
+  
+  def generate_input_init(type) 
+    text = "public init("
+      input_fields = type.required_input_fields + type.optional_input_fields
+      input_fields.each do |field|
+        text << escape_reserved_word(field.camelize_name)
+        text << ": "
+        text << swift_input_type(field.type)
+        text << (field.type.non_null? ? "" : " = nil")
+        text << (field == input_fields.last ? "" : ", ")
+      end
+    text << ")"
+    text << " {\n"
+      type.input_fields.each do |field|
+        name = escape_reserved_word(field.camelize_name)
+        text << "self." + name + " = " + name
+        text << "\n"
+      end
+    text << "}"
+  end
+  
+  def remove_linebreaks(text)
+    text.gsub("\n", " ")
+  end
+  
+  def input_field_description(type)
+    unless type.input_fields.count == 0
+      text = "/// - parameters:" + ""
+      type.input_fields.each do |field|
+        description = (field.description.nil? ? "No description" : remove_linebreaks(field.description))
+        text << "\n///     - " + field.name + ": " + description
+      end
+      text << "\n///"
+      text
+    end
+  end
+  
   def swift_arg_defs(field)
-    defs = ["aliasSuffix: String? = nil"]
+    defs = ["alias: String? = nil"]
```

Comment:
These "minor renames" are actually breaking changes.  Also, the reason why it isn't just called `alias` is because we prefix what is passed in with the field name so that there aren't conflicts between fields.  By just calling it `alias` it makes it sound like it is used literally as the GraphQL alias, which makes it seem like the caller needs to worry about preventing conflicts with other fields of the same name.